deepspeech/exps/u2/model.py

```diff
@@ -262,14 +266,14 @@ def setup_dataloader(self):
         # config.data.max_output_len = float('inf')  # tokens
         # config.data.min_output_input_ratio = 0.00
         # config.data.max_output_input_ratio = float('inf')
-        test_dataset = ManifestDataset.from_config(config)
+        test_dataset = Dataset.from_config(config)
         # return text ord id
         self.test_loader = DataLoader(
             test_dataset,
             batch_size=config.decoding.batch_size,
             shuffle=False,
             drop_last=False,
-            collate_fn=SpeechCollator(keep_transcription_text=True))
+            collate_fn=SpeechCollator(keep_transcription_text=False))
```

Comment:
keep_transcription_text必须是True

Reply:
yes, I understood. I changed this one for evaluating bleu since I read 'ids' from the preprocessed data. I will modify the code to be consistent with the current data pipeline.